WBooks/src/app/screens/BookDetails/components/BookComments/index.js

```diff
@@ -0,0 +1,41 @@
+import React, { PureComponent } from 'react';
+import { TouchableOpacity, Text } from 'react-native';
+
+import comments from '../../../../../constants/comments';
+
+import CommentItem from './components/CommentItem';
+import styles from './styles';
+
+const INITIAL_SHOWN_COMMENTS = 2;
+
+class BookComments extends PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      showAll: false
+    };
+  }
+
+  renderComment = item => <CommentItem key={this.elementKeyExtractor(item)} comment={item} />;
+
+  elementKeyExtractor = item => item.id.toString();
```

Comment:
use string literals instead of.

```
`${item.id}`
```

Reply:
in general we use it to interpolate strings and other things.
In this case is not a problem, just for the practice.